animation_nodes/nodes/gpencil/gp_material_output.py

```diff
@@ -0,0 +1,51 @@
+import bpy
+from bpy.props import *
+from ... events import propertyChanged
+from ... base_types import AnimationNode
+
+modeTypeItems = [
+    ("LINE", "Line", "", "NONE", 0),
+    ("DOTS", "Dots", "", "NONE", 1),
+    ("BOX", "Boxes", "", "NONE", 2)
+]
+
+class GPMaterialOutputNode(bpy.types.Node, AnimationNode):
+    bl_idname = "an_GPMaterialOutputNode"
+    bl_label = "GP Material Output"
+
+    modeType: EnumProperty(name = "Mode Type", default = "LINE",
+        items = modeTypeItems, update = AnimationNode.refresh)
+
+    strokeBool: BoolProperty(name="Stroke",
+                        description = "Stroke Mode for material",
+                        default=True, update=propertyChanged)
+    fillBool: BoolProperty(name="Fill",
+                        description = "Fill Mode for material",
+                        default=False, update=propertyChanged)
```

Comment:
This should be inputs, not properties. Also, note the spaces around the equal sign.

Reply:
Changed.